ui/imports/shared/popups/qmldir

```diff
@@ -22,3 +22,4 @@ ImageCropWorkflow 1.0 ImageCropWorkflow.qml
 ImportCommunityPopup 1.0 ImportCommunityPopup.qml
 DisplayNamePopup 1.0 DisplayNamePopup.qml
 AddEditSavedAddressPopup 1.0 AddEditSavedAddressPopup.qml
+SendContactRequestModal 1.0 SendContactRequestModal.qml
```

Comment:
Again, this file seems to be (physically) duplicated in 2 different subdirectories; please remove one